pkg/apiserver/authentication/identityprovider/github/github_test.go

```diff
@@ -102,7 +103,7 @@ scopes:
 					TokenURL:    tokenURL,
 					UserInfoURL: userInfoURL,
 				},
-				RedirectURL: "http://ks-console/oauth/redirect",
+				RedirectURL: "https://console.kubesphere.io/oauth/redirect/github",
```

Comment:
Please don't use `https://console.kubesphere.io` in test code.

Reply:
fixed